### ISSUE TITLE
onupdate in db migrations feature req.

### ISSUE THREAD

**Issue by ChangePlaces** [phalcon/cphalcon#12997]
The ability to support `on update` for columns is important, especially for datetime fields

e.g. `CREATE TABLE foo(id INTEGER AUTO_INCREMENT PRIMARY KEY, bar DATETIME ON UPDATE CURRENT_TIMESTAMP)`

Will this be supported?

**Comment by ChangePlaces** [comment]
Or, it's often because the project has contibuters who don't contribute, or has no community

**Comment by niden** [comment]
@ChangePlaces Yes that is correct. But taking into account our small team, we need to ensure that we focus on non stale issues, this is why we implemented this bot.

Thank you!

**Comment by Hoet** [comment]
On an extra note regarding a model save and update function

I use phinx to set-up my tables in a new project. In most of the database defenition I use a phinx function to create the created_at and updated_at columns. I made sure that the created_at column will be filled with the current timestamp on create and that the updated_at column will be updated when the row is changed. I tested this with Navicat (database manage tool, like PHPmyAdmin). Timestamps get set correctly.

Now when I create and update/save my model within phalcon I notice an issue with the save/update function. If I create a new model and save it ($model->save()) I get a new row with the current timestamp set for the created_at column. However when I update that model and them save or update it the updated_at column is not set to the current timestamp.

I can ofcourse use the 'beforeValidationOnUpdate' to update the field manually, buy I should not have to as the database column is set-up to do that on a update of the row in question.

```
    // todo: this should not be needed as the database has an update set on the column
    protected function beforeValidationOnUpdate()
    {
      $this->updated_at =date( "Y-m-d H:i:s", time( ));
    }
```

So I'm wondering why is the updated_at on a row not change when I update that row ($model->update()) or save it ($model->save()).
